utils/netutils/netutils.go

```diff
@@ -213,6 +213,23 @@ func ParseTagRanges(ranges string, tagType string) ([]TagRange, error) {
 	return tagRanges, nil
 }
 
+// IsTagInRange verifies if the passed pktTag adheres to the valueRange
+func IsTagInRange(pktTag int, valueRangeStr string, encap string) bool {
+	// Parse the valueRange to form a []TagRange object
+	valueRanges, err := ParseTagRanges(valueRangeStr, "vlan")
+	if err != nil {
+		log.Errorf("Not able to pass valueRange. Err: %s", err)
+		return false
+	}
+
+	for _, valueRange := range valueRanges {
```

Comment:
@jainvipin a lot of places where we would parse tagRange, we won't have the Tag information yet. Like in the earlier example, CreateTenant will parse tagRange, but not have pktTag info yet. That would come later when netctl net create happens.. Am I missing something?